pkg/volume/azure_file/azure_file.go

```diff
@@ -21,17 +21,19 @@ import (
 	"os"
 	"runtime"
 
+	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
nit: group import with prefix, e.g. `k8s.io` and others

Reply:
it's made by IDE automatically...

Reply:
if remove the empty lines, gofmt will auto do the sort.